src/pages/stories/Stories.tsx

```diff
@@ -30,18 +29,11 @@ type StoryListView = {
 };
 
 const Stories: React.FC = () => {
-  // const [user, setUser] = useState<string>('');
   const [data, setData] = useState<StoryListView[]>([]);
-  // const [query, setQuery] = useState('');
+  const [query, setQuery] = useState<string>('');
```

Comment:
We don't have to manually specify the type, we can use type inference;

```suggestion
  const [query, setQuery] = useState('');
```